open-company/open-company-web
# Fix menu separators

To test:
- check the menu on desktop with an admin
- check the menu on desktop with a contributor
- check the menu on desktop with a viewer
- check the menu on desktop with a viewer with at least 1 private section edit (adds reminders item)
- repeat on mobile for all
